processing/src/main/java/org/apache/druid/query/filter/DimFilter.java

```diff
@@ -53,7 +53,8 @@
     @JsonSubTypes.Type(name = "equals", value = EqualityFilter.class),
     @JsonSubTypes.Type(name = "range", value = RangeFilter.class),
     @JsonSubTypes.Type(name = "isfalse", value = IsFalseDimFilter.class),
-    @JsonSubTypes.Type(name = "istrue", value = IsTrueDimFilter.class)
+    @JsonSubTypes.Type(name = "istrue", value = IsTrueDimFilter.class),
+    @JsonSubTypes.Type(name = "arrayContainsElement", value = ArrayContainsElementFilter.class)
```

Comment:
I didn't want to confuse this native filter with the array_contains function which this filter can be used to implement but doesn't completely implement (see PR description), so I'd prefer to leave it like this to avoid confusion